app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -348,7 +351,7 @@ def redirect_path_for_switch_language
       end
 
       def redirect_path_after_create_page
-        if @page.redirects_to_external?
+        if @page.redirects_to_external? || !@page.editable_by?(current_alchemy_user)
```

Comment:
Now we have to conditionals that result in a `redirect_to admin_pages_path`, but only one test for it. I'm sorry to be so picky, but I still don't get which branch gets hit under what condition. Can I ask you for a second spec that would elucidate both conditions?